phoenix-core/src/main/java/org/apache/phoenix/schema/PSchemaKey.java

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
It is not needed anymore as we are storing schema in SYSTEM.CATALOG only. So I have removed in latest commit